code/game/objects/items/robot/robot_parts.dm

```diff
@@ -210,7 +210,6 @@
 			O.invisibility = 0
 			//Transfer debug settings to new mob
 			O.custom_name = created_name
-			O.updatename("Default")

```

Comment:
updatename is already called below, and that proc doesn't even use any argument anymore.